src/structure/immutable/deepEqual.js

```diff
@@ -1,20 +1,22 @@
 import { Iterable } from 'immutable'
-import every from '../../util/every'
 
-const deepEqualValues = (a, b) => {
-  if (a === undefined && b === '') return true
-  if (a === '' && b === undefined) return true
-  if (Iterable.isIterable(a)) {
-    if (Iterable.isIterable(b)) {
-      return a.count() === b.count() &&
-        a.every((value, key) => deepEqualValues(value, b.get(key)))
-    }
-    return false
-  }
-  if (!a || !b || typeof a != 'object' && typeof b != 'object') {
-    return a === b
+import isEqualWith from 'lodash.isEqualWith'
+
+const customizer = (obj, other) => {
+  if (obj === undefined && other === '') return true
```

Comment:
> Is `null` also something you want to lump together?

I suspect so. Thanks, I think that's a bug.

> If so an `obj == null` check will capture `null` and `undefined`.

Yes, I'm aware of that, but I prefer to use `===` whenever I can to be explicit. I think a linter beat that into me a few years ago, and now I [punish all the other monkeys](http://i.snag.gy/kdu77.jpg).